networkit/graphio.pyx

```diff
@@ -1712,15 +1720,16 @@ class GEXFReader:
 	def createEvent(self, eventTime, eventType, u, v, w):
 		"""
 		createEvent(eventTime, eventType, u, v, w)
+
 		Creates a NetworKit::GraphEvent from the supplied parameters
 		and passes it to eventStream.
+
 		Parameters
 		----------
 		eventTime : int
 			Timestep indicating when the event happen (creating an order of events).
 		eventType : str
-			Abbreviation string representing a graph event. Should be one of the following:
-			:code:`e, an, dn, rn, ae, re, de, ce`.
+			Abbreviation string representing a graph event. Should be one of the following: {`e, an, dn, rn, ae, re, de, ce`.}
```

Comment:
Does this docstring looks as intended after the change? I know currently is looks broken, however normally `:code:` works in sphynx docs.